docs-source/content/release-notes.md

```diff
@@ -8,7 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities | Change |
 | --- | --- | --- | --- |
-| TBD | v2.2 | no  | TBD 
+| TBD | v2.2 | no  | Add support for FMW Infrastructure domains. If servers are manually shutdown, for example using the console, the operator will return them to the desired state specified in the domain custom resource.  Operator is now built and runs on JDK 11. 
```

Comment:
Add -> Added
using the console, -> using the WLS Console,

Reply:
fixed